packages/coinstac-ui/app/render/components/projects/form-project-controller.js

```diff
@@ -97,6 +86,51 @@ class FormProjectController extends Component {
     });
   }
 
+  /**
+   * Get errors from the project state.
+   *
+   * @param {Object} project
+   * @returns {Object}
+   */
+  getErrors(project) {
+    const {
+      consortia,
+    } = this.props;
+
+    return Object.keys(project).reduce((memo, key) => {
+      const value = project[key];
+
+      if (key === 'metaCovariateMapping') {
```

Comment:
`getErrors` is no longer static because we need the active consortium for error checks in the metadata-to-covariate mapping.